src/main/java/org/molgenis/vcf/utils/model/FieldImpl.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.vcf.utils.model;
+
+import java.util.Set;
+import lombok.Builder.Default;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+
+@Data
```

Comment:
Immutable `@Value` class would be better?

Reply:
Doesn't work properly in combination with extending this class (NestedField)